apps/sms/js/thread_ui.js

```diff
@@ -2074,8 +2059,8 @@ var ThreadUI = {
           params.items.push({
             l10nId: 'forward',
             method: () => {
-              this.navigateToComposer({
-                forward: { messageId: messageId }
+              MessageManager.getMessage(messageId).then((message) => {
```

Comment:
I expect that the "getMessage" API is quick anyway :) it ias a DB behind the scene and id is indexed :)